src/Tribe/Aggregator/Processes/Batch_Imports.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * Class Tribe__Events__Aggregator__Processes__Batch_Imports
+ *
+ * @since TBD
+ */
+
+/**
+ * Class Tribe__Events__Aggregator__Processes__Batch_Imports
+ *
+ * Add custom hooks in order to support batch pushing.
+ *
+ * @since TBD
+ */
+class Tribe__Events__Aggregator__Processes__Batch_Imports {
+	/**
+	 * Attach hooks to support batch pushing.
+	 *
+	 * @since TBD
+	 */
+	public function hook() {
+		add_filter( 'tribe_events_aggregator_build_url', [ $this, 'build_url' ], 10, 3 );
+		add_filter( 'tribe_aggregator_service_post_import_args', [ $this, 'import_args' ], 10, 2 );
+		add_filter( 'tribe_aggregator_allow_batch_push', [ $this, 'allow_batch_import' ], 10, 2 );
+	}
+
+	/**
+	 * Update the endpoint used to initiate an process an import of events.
+	 *
+	 * @param string   $url      The completed URL generated.
+	 * @param string   $endpoint The path of the endpoint inside of the base url.
+	 * @param stdClass $api      An object representing the properties of the API.
+	 *
+	 * @return string The modified URL where to hit to process an import.
+	 */
+	public function build_url( $url, $endpoint, $api ) {
+		if ( 'import' !== $endpoint ) {
+			return $url;
+		}
+
+		return "{$api->domain}{$api->path}{v2.0.0}/{$endpoint}";
+	}
+
+	/**
+	 * Filter subsequent imports to mark children imports as batch pushing if the parent was marked as batch pushing or
```

Comment:
This description is misleading in implying a db side effect but really checking the db to return a boolean.
Base the description on what the method does, not where it's used and to which end, as it seems to be the case here.